perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -2089,7 +2089,7 @@ public void deleteAttribute(PerunSession sess, AttributeDefinition attribute) th
 	public void deleteAllAttributeAuthz(PerunSession sess, AttributeDefinition attribute) throws InternalErrorException {
 		try {
 			if (0 < jdbc.update("DELETE FROM attributes_authz WHERE attr_id=?", attribute.getId())) {
-				log.info("All attribute_authz were deleted for Attribute={}.", attribute);
+				log.debug("All attribute_authz were deleted for Attribute={}.", attribute);
```

Comment:
Since you log the whole object you can change ending to ``deleted for {}.``, since attributes ``toString()`` is like ``Attribute:[content]``.